genon/preprocessor/facade/legacy/BOK_적재용_규정.py

```
@@ -0,0 +1,1974 @@
+                    # (너무 작은 bbox일 경우 0으로 나누는 걸 방지)
+                    zoom_factor = target_height / bbox_height if bbox_height > 0 else 1.0
+                    zoom_factor = min(zoom_factor, 4.0)  # 최대 확대 비율 제한
+                    zoom_factor = max(zoom_factor, 1)  # 최소 확대 비율 제한
+
+                    # 페이지를 이미지로 렌더링
+                    mat = fitz.Matrix(zoom_factor, zoom_factor)
+                    pix = page.get_pixmap(matrix=mat, clip=cell_bbox)
+                    img_data = pix.tobytes("png")
+
+                    result = post_ocr_bytes(img_data, timeout=OCR_TABLE_CELL_TIMEOUT)
+                    rec_texts, rec_scores, rec_boxes = extract_ocr_fields(result)
```

Ruff (0.15.18)

[warning] Unpacked variable `rec_scores` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)